messages/en.json

```diff
@@ -84,7 +93,12 @@
         "A pack of wild animals sets upon \u0002{0}\u0002. Soon they are only a pile of bones and a lump in the beasts' stomachs.",
         "\u0002{0}\u0002 fell off the roof of their house and is now dead.",
         "\u0002{0}\u0002 is crushed to death by a falling tree. The villagers desperately try to save them, but it is too late.",
-        "\u0002{0}\u0002 suddenly bursts into flames and is now all but a memory."
+        "\u0002{0}\u0002 suddenly bursts into flames and is now all but a memory.",
+        "\u0002{0}\u0002 played too much Russian roulette with their so called friends, and lost. The villagers bury the leftovers.",
```

Comment:
Which parts? The whole thing?
What about something like "played Russian roulette with their so called friends, and lost." Instead?